easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -125,11 +125,13 @@ public static void requestPermissions(@NonNull final Object object, @NonNull Str
      *         request code to track this request, must be < 256.
      * @param perms
      *         a set of permissions to be requested.
+     * @return true if a permissions request was started, false otherwise.
      */
     @SuppressLint("NewApi")
-    public static void requestPermissions(@NonNull final Object object, @NonNull String rationale,
+    public static boolean requestPermissions(@NonNull final Object object, @NonNull String rationale,
```

Comment:
I meant to include checking for permissions in the if statement (I fixed that in my latest commit). This leaves us with two questions: should we be checking for permissions as well and then does it make sense to return the boolean if we are also checking for permissions?